### DISCUSSION TITLE
https://starter.obytes.com/ci-cd/workflows-references/

### DISCUSSION THREAD

**Discussion by giscus** [obytes/react-native-template-obytes#290] [Docs Comments]
# https://starter.obytes.com/ci-cd/workflows-references/

A list of all the Github Action workflows that come with the starter and how to use them.

https://starter.obytes.com/ci-cd/workflows-references/

**Comment by Inalegwu** [comment]
How can I make environment variables available to eas ?

**Reply by yjose** [reply]
check this guide here https://docs.expo.dev/build-reference/variables/#secrets-on-the-expo-website

**Comment by OgDev-01** [comment]
What if i want to use main branch as my production branch instead of master?... do i need to configure additional steps aside deleting master branch, and removing the references in all actions?

**Reply by yjose** [reply]
Yes, this will require updating the workflow to dispatch rules based on the conditions and branches you want.
